Yin/VideoOperation.swift

```diff
@@ -39,6 +39,16 @@ enum VideoOperation {
             completion(true, nil)
 
             //This is how you catch error let error!! sigh
+            // I recently learned that swift automatically adds the variable named `error` in `catch` blocks, which means you actually don't need to write out `let error` unless you want to \o/.
```

Comment:
How do you know? Could you link me the reference please ?

Reply:
According to apple "You write a pattern after catch to indicate what errors that clause can handle. If a catch clause doesn’t have a pattern, the clause matches any error and binds the error to a local constant named error. For more information about pattern matching, see Patterns."


It would firstly prioritize otherError if it throws error, if it does not find other Error, then it would check some error, I believe.